test/test_nn.py

```diff
@@ -10717,6 +10683,44 @@ def test_contig_wrong_stride_cudnn(self, device):
         F.conv_transpose2d(x, torch.randn(16, 1, 1, 1, device=device))
         F.conv2d(x, torch.randn(1, 16, 1, 1, device=device))
 
+    @onlyCUDA
+    def test_Conv2d_size_1_kernel(self, device):
+        x_cpu = torch.randn(2, 3, 5, 5, requires_grad=True)
```

Comment:
since you are not checking x_cpu.grad anyway, requires_grad=True is not required.